x-pack/plugin/esql/qa/testFixtures/src/main/resources/date.csv-spec

```diff
@@ -139,9 +139,9 @@ emp_no:integer | birth_date:date               | x:date
 ;
 
 evalDateTruncGrouping
-from employees | eval y = date_trunc(1 year, hire_date) | stats count(emp_no) by y | sort y | keep y, count(emp_no) | limit 5;
+from employees | eval y = date_trunc(1 year, hire_date) | stats c = count(emp_no) by y | sort y | keep y, c | limit 5;
 
-y:date                        | count(emp_no):long
+y:date                        | c:long
```

Comment:
A side effect of this change - the expression has to be quoted or an alias used.

Reply:
I think this is a positive side effect; being able to use an unquoted function-like alias seems inconsistent